Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
+    console.log(a + b + c)
+}
+info(1, 5, 34);
+
+//2
+
+function colorCar(z) {
+    print = `a ${z} car`
+    console.log(print);
+}
+colorCar("red");
+
+//3
+
+let obj = {
```

Comment:
Do you think it's better to use `let` here, or `const`?

Reply:
If I would like to change, it can be let. If I wouldn't, not necessary.